### PR TITLE
Sorting algorithms

### PR DESCRIPTION
**Pull request by Rameshchandrapola**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/Tenet-Coding/ArtofCoding/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added four sorting algorithms.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #11

### CODE REVIEW

File: C++/Sorting/bubble_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    Time Complexity  : O(N^2) 
+    Space Complexity : O(1)
+    
+Sample Input :
+     Enter the number of elements in the array : 7
+     Enter the values of numbers in the array:
+     7 4 1 5 2 6 3
+Output :
+    Sorted array: 
+        1 2 3 4 5 6 7
```
Comment (Harshil-Jani) [inline C++/Sorting/bubble_sort.cpp:55]: Add new line at EOF.

File: C++/Sorting/merge_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    Space Complexity : O(N)
+    
+Sample Input :
+     Enter the number of elements in the array : 7
+     Enter the values of numbers in the array:
+     7 4 1 5 2 6 3
+Output :
+    Sorted array: 
+        1 2 3 4 5 6 7
+*/
```
Comment (Harshil-Jani) [inline C++/Sorting/merge_sort.cpp:99]: Add new line at EOF.

File: C++/Sorting/selection_sort.cpp [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    Space Complexity : O(1)
+    
+Sample Input :
+     Enter the number of elements in the array : 7
+     Enter the values of numbers in the array:
+     7 4 1 5 2 6 3
+Output :
+    Sorted array: 
+        1 2 3 4 5 6 7
+*/
```
Comment (Harshil-Jani) [inline C++/Sorting/selection_sort.cpp:58]: Add new line at EOF.

**Review by Harshil-Jani — COMMENTED** [review]
Just add new line at End of Files.

File: C++/Sorting/bubble_sort.cpp [hunk 4]
```diff
@@ -0,0 +1,56 @@
+#include <bits/stdc++.h>
```
Comment (soumik2012) [inline C++/Sorting/bubble_sort.cpp:1]: Add the Description to the Problem

File: C++/Sorting/merge_sort.cpp [hunk 5]
```diff
@@ -0,0 +1,99 @@
+#include <bits/stdc++.h>
```
Comment (soumik2012) [inline C++/Sorting/merge_sort.cpp:1]: Add the Description to the Problem

**Review by soumik2012 — CHANGES_REQUESTED** [review]

**Review by soumik2012 — APPROVED** [review]
LGTM 💯

**Review by Harshil-Jani — COMMENTED** [review]
Perfect.. Everything Fine. :+1:

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**